scripts/iso-manager.sh

```diff
@@ -84,6 +90,12 @@ _add_archives() {
     done
 }
 
+_remove_archives() {
+    for archive; do
+        $SALT_CALL metalk8s.configure_archive "$archive" remove=True
```

Comment:
Not really due to this PR but it's really sad that even when you try to remove an archive, we need this one to be there and available 

IMO this line in the module shouldn't be there when we remove 
```python
    # raise if the archive does not exist or is invalid
    archive_info_from_product_txt(archive)
```